### ISSUE TITLE
Add Conf value for story sort on index page

### ISSUE THREAD

**Issue by leegarner** [glFusion/glfusion#256]
Similar to the topic settings, have a global "sort_by" and "sort_dir" for stories on the index page. This would be used when topic=0.

The change to index.php would look something like this:
```
@@ -151,6 +151,9 @@ if ( !empty($topic) ) {
     if ( $result ) {
         list($topiclimit, $story_sort, $story_sort_dir) = DB_fetchArray($result
     }
+} else {
+    $story_sort = $_CONF['story_sort_by'];
+    $story_sort_dir = $_CONF['story_sort_dir'];
 }

 $maxstories = 0;
@@ -247,7 +250,6 @@ if ($_CONF['allow_user_photo'] == 1) {
     }
 }

-if ( !empty($topic) ) {
     switch ( $story_sort ) {
         case 0 :    // date
             $orderBy = ' date ';
@@ -273,9 +275,6 @@ if ( !empty($topic) ) {
             $orderBy = $orderBy . ' DESC ';
             break;
     }
-} else {
-    $orderBy = ' date DESC';
-}
```

**Comment by mark0263** [comment]
Implemented in LTS branch with commit e7c7004e17026530b70cc2241d5f0145f242e886

**Comment by mark0263** [comment]
Implemented in develop branch with commit ac7dbd87f3e693ce052978fadb0c82412ffaba0a
